pkg/server/api/agent/v1/service.go

```diff
@@ -75,13 +75,15 @@ func (s *Service) CountAgents(ctx context.Context, req *agentv1.CountAgentsReque
 		log := rpccontext.Logger(ctx)
 		return nil, api.MakeErr(log, codes.Internal, "failed to count agents", err)
 	}
+	rpccontext.AuditRPC(ctx)
 
 	return &agentv1.CountAgentsResponse{Count: count}, nil
 }
 
 // ListAgents returns an optionally filtered and/or paginated list of agents.
 func (s *Service) ListAgents(ctx context.Context, req *agentv1.ListAgentsRequest) (*agentv1.ListAgentsResponse, error) {
 	log := rpccontext.Logger(ctx)
+	// TODO: ADD filters fields
```

Comment:
I wanted to confirm we want to add those files before implementing, if you are ok with add them I'll add those fields now

Reply:
Yeah I think these are useful to include since it reveals some information about how the API is being accessed - e.g. are clients always requesting everything, what kinds of filters are commonly used, etc.